src/openrct2/Cheats.cpp

```diff
@@ -374,10 +374,14 @@ static void cheat_remove_all_guests()
             while (spriteIndex != SPRITE_INDEX_NULL)
             {
                 vehicle = GET_VEHICLE(spriteIndex);
-
+                size_t offset = 0;
                 for (size_t i = 0; i < vehicle->num_peeps; i++) 
```

Comment:
`for (size_t i = 0, offset = 0; i < vehicle->num_peeps; i++) `

Reply:
OK, didn't know that syntax 😁